### PR TITLE
[BugFix] Fix Max/Min aggregate function column statistics error

### LINKED ISSUE #5977: Max/Min column statistics compute error
<!-- (At least include the following, feel free to add if you have more content) -->

### Steps to reproduce the behavior (Required)
```
mysql> explain costs select Min(C_NAME) as min, Max(C_NAME) as max from customer having max < '1234';
+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| Explain String                                                                                                                                                                                                                                   |
+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| PLAN FRAGMENT 0(F01)                                                                                                                                                                                                                             |
|   Output Exprs:9: min | 10: max                                                                                                                                                                                                                  |
|   Input Partition: UNPARTITIONED                                                                                                                                                                                                                 |
|   RESULT SINK                                                                                                                                                                                                                                    |
|                                                                                                                                                                                                                                                  |
|   2:AGGREGATE (update finalize)                                                                                                                                                                                                                  |
|   |  aggregate: min[([2: C_NAME, VARCHAR, false]); args: VARCHAR; result: VARCHAR; args nullable: false; result nullable: true], max[([2: C_NAME, VARCHAR, false]); args: VARCHAR; result: VARCHAR; args nullable: false; result nullable: true] |
|   |  having: [10: max, VARCHAR, true] < '1234'                                                                                                                                                                                                   |
|   |  cardinality: 0                                                                                                                                                                                                                              |
|   |  column statistics:                                                                                                                                                                                                                          |
|   |  * min-->[-Infinity, -Infinity, 0.0, 16.0, 1.0] ESTIMATE                                                                                                                                                                                     |
|   |  * max-->[Infinity, Infinity, 0.0, 16.0, 1.0] ESTIMATE                                                                                                                                                                                       |
|   |                                                                                                                                                                                                                                              |
|   1:EXCHANGE                                                                                                                                                                                                                                     |
|      cardinality: 15000000                                                                                                                                                                                                                       |
|                                                                                                                                                                                                                                                  |
| PLAN FRAGMENT 1(F00)                                                                                                                                                                                                                             |
|                                                                                                                                                                                                                                                  |
|   Input Partition: RANDOM                                                                                                                                                                                                                        |
|   OutPut Partition: UNPARTITIONED                                                                                                                                                                                                                |
|   OutPut Exchange Id: 01                                                                                                                                                                                                                         |
|                                                                                                                                                                                                                                                  |
|   0:OlapScanNode                                                                                                                                                                                                                                 |
|      table: customer, rollup: customer                                                                                                                                                                                                           |
|      preAggregation: on                                                                                                                                                                                                                          |
|      partitionsRatio=1/1, tabletsRatio=12/12                                                                                                                                                                                                     |
|      tabletList=1565205,1565207,1565209,1565211,1565213,1565215,1565217,1565219,1565221,1565223 ...                                                                                                                                              |
|      actualRows=15000000, avgRowSize=18.0                                                                                                                                                                                                        |
|      cardinality: 15000000                                                                                                                                                                                                                       |
|      column statistics:                                                                                                                                                                                                                          |
|      * C_NAME-->[-Infinity, Infinity, 0.0, 18.0, 1.5E7] ESTIMATE                                                                                                                                                                                 |
+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
31 rows in set (0.00 sec)
```

### Expected behavior (Required)
give right cardinality estimate

### Real behavior (Required)

### StarRocks version (Required)
 - You can get the StarRocks version by executing SQL `select current_version()`

### PR DESCRIPTION
**Pull request by Youngwb**
## What type of PR is this：
- [x] bug
- [ ] feature
- [ ] enhancement
- [ ] others

## Which issues of this PR fixes ：

Fixes #5977 

## Problem Summary(Required) ：

The minValue, maxValue and NDV of column statistics for Min/Max function has computed error,  and also fix sum function

### CODE REVIEW

File: fe/fe-core/src/main/java/com/starrocks/sql/optimizer/statistics/ExpressionStatisticCalculator.java [hunk 1]
```diff
@@ -42,7 +42,7 @@ private static class ExpressionStatisticVisitor extends ScalarOperatorVisitor<Co
 
         public ExpressionStatisticVisitor(Statistics statistics, double rowCount) {
             this.inputStatistics = statistics;
-            this.rowCount = rowCount;
+            this.rowCount = Math.max(1.0, rowCount);
```
Comment (chaoyli) [inline fe/fe-core/src/main/java/com/starrocks/sql/optimizer/statistics/ExpressionStatisticCalculator.java:45]: Please obey the rule of the pull request and commit message.

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 25 / 26 (96.15%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/optimizer/statistics/StatisticRangeValues.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/sql/optimizer/statistics/PredicateStatisticsCalculator.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/sql/optimizer/statistics/ExpressionStatisticCalculator.java|24|24|100.00%|

**Review by kangkaisen — APPROVED** [review]

**Review by HangyuanLiu — APPROVED** [review]

**Comment by Youngwb** [comment]
@MergifyIO backport branch-2.2

**Comment by Youngwb** [comment]
@Mergifyio backport branch-2.1

**Pull request merged**